extras/upgrade_activities.sh

```diff
@@ -29,6 +29,9 @@ function my_exit()
   if [[ ${GBD_STARTED} -ne 0 ]]
   then
     pkill -15 gluster-blockd
+
+    # let's wait until the child process is finished 
+    wait
```

Comment:
Xiubo, should we rather be more specific ?
wait \<pid-of-gluster-blockd\> ?

Reply:
This \<pid-of-gluster-blockd\> need to be captured after:

```
# start gluster-blockd                                                                                                                                                                                             
gluster-blockd --no-remote-rpc & > /dev/null 2>&1
GB_PID=${!}
```

Else we might endup getting pidof actual gluster-blockd started by systemd, may be ?

Reply:
This is okay.

The "wait" here will only wait the child processes running in background of this script, here is only the "gluster-blockd --no-remote-rpc &".

But I am okay with adding the PID.